ce-sample-post-authn-api-go/README.md

```diff
@@ -0,0 +1,123 @@
+## ce-samples-post-authn-api-go
+
+This module contains the server side API code written in GoLang.
+
+### Functionality
+
+- Provide server side authentication with ACP
+- Package up ACP session and customer API response and return it to UI
+- Stubbed out call to customer API
+- Provide ACP getSession, completePostAuthn, abortPostAuthn
+- Keep ACP and customer API credentials secret
+
+### Configure Go
+
+Create the Go source directory
+
+```
+mkdir ~/go/src/github.com
+```
+
+Add the following environment variables to your `~/.bash_profile`
+
+```
+export GOPRIVATE=github.com/cloudentity/*
+export GOROOT=/usr/local/go
+export GOPROXY=https://proxy.golang.org,direct
+export GO111MODULE=on
+export GOPATH=$HOME/go
+```
+
+Be sure to start a new shell so that the new environment variables are set.
+
+### Clone the repo
+
+```
+cd ~/go/src/github.com
+```
+
+```
+git clone git@github.com:cloudentity/ce-samples-post-authn.git
+```
+
+### Configure the IssuerUrl
+
+In the ACP UI, get the `Issurer URL` from your application configuration 
+
+https://cloudentity.com/developers/howtos/extensions/custom_apps/#register-application-in-cloudentity
+
+In the `config/config.go` file, copy the `Issurer URL` into the `IssuerUrl`.
+
+```
+const IssuerUrl string = "https://[TENANT_ID].us.authz.cloudentity.io/[TENANT_ID]/system/oauth2/token"
```

Comment:
I think you meant to have https://[TENANT_ID].us.authz.cloudentity.io/[TENANT_ID]/system here, since the token path is appended via the AuthnUrl value

Reply:
Good catch.  I fixed it.